SC data release notes 21

pmb59 opened the pull request:
Starting release notes for September '24 SCXA release.

pcm32 committed:
Initial commit for SC data release notes 21

anilthanki committed:
Update 2024-09-14-21.md - adds reference version

anilthanki commented on release-notes/sc/_posts/2024-09-14-21.md:
```diff
@@ -0,0 +1,38 @@
+- This release contains **381** single cell RNA-Seq studies, consisting of **24,323,556** cells, of which **13,519,191** passed our QC from **21** different species.
+
+#### New experiments
+- [Human Lung Cell Atlas: An integrated cell atlas of the human lung in health and disease](https://www.ebi.ac.uk/gxa/sc/experiments/E-ANND-1)
```
```suggestion
Externally analysed dataset
- [Human Lung Cell Atlas: An integrated cell atlas of the human lung in health and disease](https://www.ebi.ac.uk/gxa/sc/experiments/E-ANND-1)
```

anilthanki commented on release-notes/sc/_posts/2024-09-14-21.md:
```diff
@@ -0,0 +1,38 @@
+- [GTEx: snRNAseq atlas](https://www.ebi.ac.uk/gxa/sc/experiments/E-ANND-2)
+- [Tabula Sapiens - Smart-seq2 dataset](https://www.ebi.ac.uk/gxa/sc/experiments/E-ANND-3)
+- [Mapping the developing human immune system across organs](https://www.ebi.ac.uk/gxa/sc/experiments/E-ANND-5)
+- [Single cell RNA-sequencing of normal and fibrotic lungs in human](https://www.ebi.ac.uk/gxa/sc/experiments/E-CURD-126)
```
```suggestion

Dataset reanalysed with our standard pipelines
- [Single cell RNA-sequencing of normal and fibrotic lungs in human](https://www.ebi.ac.uk/gxa/sc/experiments/E-CURD-126)
```

pmb59 committed:
Update release-notes/sc/_posts/2024-09-14-21.md

Co-authored-by: Anil Thanki <thanki.anil@gmail.com>

pmb59 committed:
Update release-notes/sc/_posts/2024-09-14-21.md

Co-authored-by: Anil Thanki <thanki.anil@gmail.com>

pmb59 committed:
add New features

anilthanki commented:
Not sure if we should mention about an experiment we dropped `E-MTAB-6818`

sfexova committed:
adding highlights

Added highlights and edited software features section

sfexova committed:
Update 2024-09-14-21.md - fixing examples for soft features

forfot to add example for ICT marker gene heatmap, edited example for gut anatomogram

lingyun1010 commented on release-notes/sc/_posts/2024-09-14-21.md:
```diff
@@ -0,0 +1,51 @@
+- This release contains **381** single cell RNA-Seq studies, consisting of **24,323,556** cells, of which **13,519,191** passed our QC from **21** different species.
+
+#### Software features
+- **Gut anatomogram** - a new human anatomogram for the large and small intestines has been added in this release and accompanies eligible experiments e.g. [here](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-125970/results/anatomogram)
```
I think we could replace `here` with the experiment accession.

lingyun1010 commented on release-notes/sc/_posts/2024-09-14-21.md:
```diff
@@ -0,0 +1,51 @@
+#### New experiments
+Highlights
+- **Externally analysed data** - for the first time in this release we have included a new class of data to Single Cell Expression Atlas - externally analysed data, marked with an 'E' icon. 
+  These are currently limited to a careful selection of eligible high-impact human datasets including the [GTEx: snRNAseq atlas](https://www.ebi.ac.uk/gxa/sc/experiments/E-ANND-2) As these    experiments are not re-analysed with our own analysis pipelines, not all the standard chart or data options may be available
```
Extra space between these and experiments

lingyun1010 commented on release-notes/sc/_posts/2024-09-14-21.md:
```diff
@@ -0,0 +1,51 @@
+
+#### Software features
+- **Gut anatomogram** - a new human anatomogram for the large and small intestines has been added in this release and accompanies eligible experiments e.g. [here](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-125970/results/anatomogram)
+- **Inferred cell type marker gene heatmap** - for experiments where inferred cell type data is available you can now view marker genes based on the inferred cell type clusters e.g. [here](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-227038/results/marker-genes)
```
Same comment for here

sfexova committed:
Update 2024-09-14-21.md - incorporating Lingyun's comments

irisdianauy commented on release-notes/sc/_posts/2024-09-14-21.md:
```diff
@@ -0,0 +1,51 @@
+
+#### New experiments
+Highlights
+- **Externally analysed data** - for the first time in this release we have included a new class of data to Single Cell Expression Atlas - externally analysed data, marked with an 'E' icon. 
```
Adding punctuations to clarify sentence
```suggestion
- **Externally analysed data** - for the first time, in this release, we have included a new class of data to Single Cell Expression Atlas - externally analysed data, marked with an 'E' icon. 
```

sfexova replied:
thank you, I re-phrased the sentence for greater clarity

irisdianauy reviewed (commented):
LGTM, aside from the small request to add punctuations.

sfexova committed:
Update 2024-09-14-21.md - adding resource links to features

Incorporating feedback from Iris and Karoly, adding multiple resource links to list of features, with an example

sfexova reviewed (approved):
Made a few edits and incorporated other ppl's feedback, looks good to me now

sfexova committed:
Update 2024-09-14-21.md - added info to Highlights

sfexova committed:
Update 2024-09-14-21.md - correcting typo

Resource > Resources as that's how the section is called on the web

sfexova committed:
Update 2024-09-14-21.md

change boldface in soft features
